lib/Command/PreGenerate.php

```diff
@@ -88,7 +88,7 @@ public function __construct(string $appName,
 	protected function configure(): void {
 		$this
 			->setName('preview:pre-generate')
-			->setDescription('Pre generate previews');
+			->setDescription('Pre generate one preview resolution per image');
```

Comment:
That is not true. The `pre-generate` command will also generate all configured preview resolutions. 

The difference is that `pre-generate` will only generate previews for files that have been uploaded or changed since the last time it was run.